pkg/tanka/list.go

```diff
@@ -42,18 +43,28 @@ func ListEnvs(dir string, opts ListOpts) ([]*v1alpha1.Environment, error) {
 }
 
 // list implements the actual functionality described at 'ListEnvs'
-func list(dir string) ([]*v1alpha1.Environment, error) {
-	// list directory, also checks if dir
-	files, err := ioutil.ReadDir(dir)
+func list(path string) ([]*v1alpha1.Environment, error) {
+	// try if this has envs
+	list, err := List(path, jsonnet.Opts{})
```

Comment:
`List`, `list` and `ListEnvs` are hard to keep mentally distinct. We should find better names for that (not necessarily in this PR), e.g. by having `List` for the loader operation, and `findEnvs` and `FindEnvs` for the searching task

Reply:
Sure, I agree. What do we call the filename then? `enviroments.go` perhaps .🤡

Reply:
`find.go`? Really doesn't matter, changed it back then to keep the `git diff` cleaner so it was easier to review for y'all